README.md

```diff
@@ -301,13 +310,99 @@ var authClient = new OktaAuth(config);
 
 ### Configuration options
 
-These configuration options can be included when instantiating Okta Auth JS (`new OktaAuth(config)`) or in `token.getWithoutPrompt`, `token.getWithPopup`, or `token.getWithRedirect` (unless noted otherwise). If included in both, the value passed in the method takes priority.
+Most configuration options can be included when instantiating Okta Auth JS (`new OktaAuth(config)`) or in `token.getWithoutPrompt`, `token.getWithPopup`, or `token.getWithRedirect` (unless noted otherwise). If included in both, the value passed in the method takes priority.
+
+**Important:** These configuration options can **only** be used when instantiating Okta Auth JS.
+
+#### `storageManager`
+
+The `storageManager` provides access to client storage for specific purposes. `storageManager` configuration is divided into named sections. The default configuration is shown below:
+
+```javascript
+
+var config = {
+  storageManager: {
+    token: {
+      storageTypes: [
```

Comment:
so here means the supported storage types?

Reply:
no, these are the preferred storage types in fallback order

the "storageUtil" (browser or server) decides what types are supported

Reply:
Perhaps a name to indicate that?   `storageOrder`?